packages/flutter_tools/lib/src/compile.dart

```diff
@@ -233,4 +237,9 @@ class ResidentCompiler {
   void reset() {
     _server.stdin.writeln('reset');
   }
+
+  Future<dynamic> shutdown() {
+    _server.kill();
```

Comment:
Do we have to kill the frontend server? Wouldn't sending 'quit' command work more cleanly?

Reply:
Yeah, `quit` is probably cleaner, but I wanted to avoid handling the case when frontend server got into some weird state - handling that would require sending `quit`, waiting with timeout for server to exit, then killing it. 

We can clean this up after we land this CL.

Reply:
Arguably, if frontend server can get into weird state so that it doesn't respond to 'quit' command, we would want to know about it now. After all this is test suite that should help us uncover bugs like that.
Cleaning it up later is fine though.